### DISCUSSION TITLE
keybinding shortcut not displaying  properly

### DISCUSSION THREAD

**Discussion by Naimul007A** [prasanthrangan/hyprdots#1949] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/1cd228cf-a448-4c57-8c1b-fc3ca206c29b)
when i installed it's showing properly  but maybe i changed something after then it's showing this.Please help how can i solved this.I run this ``` ./install.sh -r``` but it's still not working

**Comment by kRHYME7** [comment]
output when you run ` keybinds_hint.sh ` ?

**Reply by Naimul007A** [reply]
> output when you run `keybinds_hint.sh` ?

![image](https://github.com/user-attachments/assets/f47ac316-4f4d-42f8-8c4b-9d72ca5cdbd0)

**Reply by Naimul007A** [reply]
i changed the default terminal kitty to warp thats why showing the problem .
Can i solved this  for warp terminal ?

**Reply by kRHYME7** [reply, marked as answer]
added extra ""  ???? 


use this format instead 

` bindd = mod,key,Launch terminal emulator,warp-terminal `
